src/Select.js

```diff
@@ -1081,7 +984,6 @@ const Select = createClass({
 		let className = classNames('Select', this.props.className, {
 			'Select--multi': this.props.multi,
 			'Select--single': !this.props.multi,
-			'is-clearable': this.props.clearable,
```

Comment:
I guess it was done by hand but against an older tree. I guess I lost the changes that were made against master since then.